README.md

```diff
@@ -855,7 +857,25 @@ create table <schema>.<table_name> (
 
 This means that all the fields in the JSON schema (representing the JSON object) **are** the "jsonb" (column) in the Postgres table.
 
-#### Saving binary data
+### Minimum PostgreSQL server version
+
+The minimum PostgreSQL server version for RMB 34.0 is 12.0.0. RMB fails a POST /_/tenant call if the actual version is lower. You may set a higher or lower requirement using [server_version_num format](https://www.postgresql.org/docs/current/runtime-config-preset.html#GUC-SERVER-VERSION-NUM):
+
+```
+public class TenantRefAPI extends TenantAPI {
+  @Validate
+  @Override
+  public void postTenant(TenantAttributes tenantAttributes, Map<String, String> headers,
+                         Handler<AsyncResult<Response>> handler, Context context)  {
+    // https://www.postgresql.org/docs/current/runtime-config-preset.html#GUC-SERVER-VERSION-NUM
+    context.putLocal("postgres_min_version_num", "100000");
+    context.putLocal("postgres_min_version", "10.0");  // human readable, for error message only
```

Comment:
Having only one setting - the string version - is more elegant and then calculate the numeric version from it.

Not a show-stopper, but would doing that calculation even be more code lines?

Reply:
The latest 9.6 version is 9.6.24 with server_version_num = 90624